### ISSUE TITLE
Any plans on migrating to Polymer 2?

### ISSUE THREAD

**Issue by EliArtist** [ergo/polymer-dragula#9]
Are there any plans on migrating polymer-dragula to Polymer 2?

**Comment by ergo** [comment]
I haven't tested yet, it doesn't work? I have router element that worked in hybrid mode just fine.

Usually elements like that are easy to support both 1.x and 2.x at same time. I'm waiting on some pr's getting merged into dragula itself to support shadow dom correctly.

https://github.com/bevacqua/dragula/pull/470
